modules/moa-create-dedicated-cluster-admins.adoc

```diff
@@ -5,10 +5,13 @@
 
 [id="moa-create-dedicated-cluster-admins.adoc"]
 = Creating dedicated-admins and cluster-admins
+As the user that created the cluster, add the `cluster-admin` user role to your account to have the maximum administrator privileges. You can create additional `cluster-admin` or `dedicated-admin` users. `Dedicated-admin` users have fewer privileges.
```

Comment:
If the role is `dedicated-admin`, I would leave it lower case. Another option would be to use the general descriptions + the roles. E.g.:
Cluster administrators, `cluster-admin`
Dedicated administrators, `dedicated-admin`